CovertInfiltration/Src/CovertInfiltration/Classes/UICovertActionsGeoscape_SlotInfo.uc

```diff
@@ -0,0 +1,84 @@
+//---------------------------------------------------------------------------------------
+//  AUTHOR:  Xymanek
+//  PURPOSE: This is a information holder for a single slot on the UICovertActionsGeoscape
+//           screen which handles both personnel and cost slots (but not both at the 
+//           same time).
+//           Separate class to simply logic in UICovertActionsGeoscape (since the info is
+//           needed in various places)
+//---------------------------------------------------------------------------------------
+//  WOTCStrategyOverhaul Team
+//---------------------------------------------------------------------------------------
+
+class UICovertActionsGeoscape_SlotInfo extends Object;
+
+var bool ShowPrefix; // Show "REQUIRED:" or "OPTIONAL:" prefix. Used for ongoing CAs
+var bool ColourDescription;
```

Comment:
Spelling? Colour -> Color, same in UpdateDescription above

Reply:
[British spelling](https://simple.wikipedia.org/wiki/Colour). Pretty sure I've used it in many other places